include/swift/AST/DiagnosticsParse.def

```diff
@@ -248,6 +248,8 @@ ERROR(computed_property_no_accessors, none,
       "%select{computed property|subscript}0 must have accessors specified", (bool))
 ERROR(expected_getset_in_protocol,none,
       "expected get or set in a protocol property", ())
+ERROR(unexpected_getset_implementation_in_protocol,none,
+      "protocol property cannot have a default implementation of %0", (StringRef))
```

Comment:
How about a slight rewording so that what we're referring to (the property getter/setter) isn't split between the beginning and end of the sentence, and to clarify that the error is not because a default implementation isn't ever allowed, but not allowed _here_:

```suggestion
      "protocol property %0 cannot have a default implementation specified here; use an extension instead", (StringRef))
```

Reply:
Thanks! changed it :)